src/status/types.nim

```diff
@@ -1,3 +1,5 @@
+{.used.}
```

Comment:
this is to fix https://github.com/status-im/status-desktop/pull/2974/files#diff-bfc3c2a1be0ad730f9e1e0db3b99f8844f7cc1804310861cd40fcf736d29067eL8